WalletWasabi.Tests/NodeBuilding/NodeBuilder.cs

```diff
@@ -41,6 +42,10 @@ private static async Task TryRemoveWorkingDirectoryAsync()
 			catch (DirectoryNotFoundException)
 			{
 			}
+			catch (Exception ex)
```

Comment:
It looks okay to me. What seems wrong is that there is an empty `catch(DirectoryNotFoundException)` two lines above when the invoked method closed in the try block cannot throw that exception.

Reply:
That empty trycatch came from copypasting NBitcoin. The exception I get sometimes is an unauthorized exception to the `.lock`. Nevertheless Dispose should not throw.